pull/context.go

```diff
@@ -30,6 +30,12 @@ type MembershipContext interface {
 
 	// IsCollaborator returns true if the user meets the desiredPerm of the given organzation's repository.
 	IsCollaborator(org, repo, user, desiredPerm string) (bool, error)
+
+	// ListTeamMembers returns the list of usernames in the given organization's team.
+	ListTeamMembers(org, team string) ([]string, error)
+
+	// ListOrganizationMembers returns the list of usernames in the given organization.
+	ListOrganizationMembers(org string) ([]string, error)
```

Comment:
style: these would better match existing functions on the context types as `TeamMembers` and `OrganizationMembers`, without the "list" action.